README.md

```diff
@@ -103,6 +103,30 @@ find(
 ): Promise<PagedResult<Schema>>;
 ```
 
+## 8.0.0 Upgrade Guide
+
+### Double encoding RQL values 
+From version 8.0.0 RQL values are now [double encoded](https://docs.extrahorizon.com/extrahorizon/additional-resources/resource-query-language-rql#double-encoding-of-special-characters) by default when using the rql builder, to support the use of special characters in rql operations.
+
+In your existing application it is likely that you have encoded values provided to the rql builder using methods such as `encodeURIComponent()`. To support the automatic double encoding of special characters, where applicable, these instances of encoding should be removed.
+- Before 8.0.0: `rqlBuilder().eq('diagnosis', encodeURIComponent('Hypertension - STAGE 1'))`
+- After 8.0.0: `rqlBuilder().eq('diagnosis', 'Hypertension - STAGE 1')`
+
+Any use of an rql builder that does not need to support special characters may remain as it is.
+
+#### Reverting double encoding for a single rql builder
+You may disable double encoding for an rql builder by setting the option `rqlBuilder({ doubleEncodeValues: false })`
+- Before 8.0.0: `rqlBuilder().eq('diagnosis', encodeURIComponent('Hypertension - STAGE 1'))`
+- After 8.0.0: `rqlBuilder({ doubleEncodeValues: false }).eq('diagnosis', encodeURIComponent('Hypertension - STAGE 1'))`
+
+#### Reverting double encoding for all rql builders
+The automatic double encoding of values can be reverted to behavior before 8.0.0 using the snippet `rqlBuilder.doubleEncodeValues = false`. This should normally exist alongside the declaration of your Extra Horizon SDK instance at the root of your application.
```

Comment:
`automatic double encoding` and `double encoding` are used a bit at random

Reply:
Double encoding is used when referencing the rql builder as it's object
Automatic double encoding is used when not referencing the object to ensure the knowledge that the user isn't in control of it.